database/seeders/DatabaseSeeder.php

```diff
@@ -31,6 +32,11 @@ public function run()
         $this->call(NewsSeeder::class);
         $this->call(EventSeeder::class);
 
+        if (App::environment('local')) {
```

Comment:
Can you include the followings as well under if condition? Thanks !

$this->call(NewsSeeder::class);
$this->call(EventSeeder::class);